static/app/views/replays/detail/focusArea.tsx

```diff
@@ -16,15 +16,17 @@ import MemoryChart from 'sentry/views/replays/detail/memoryChart';
 import NetworkList from 'sentry/views/replays/detail/network';
 import Trace from 'sentry/views/replays/detail/trace';
 
-type Props = {};
+type Props = {
+  width: number;
+};
 
 function getBreadcrumbsByCategory(breadcrumbs: Crumb[], categories: string[]) {
   return breadcrumbs
     .filter(isBreadcrumbTypeDefault)
     .filter(breadcrumb => categories.includes(breadcrumb.category || ''));
 }
 
-function FocusArea({}: Props) {
+function FocusArea({width = 0}: Props) {
```

Comment:
The `Props` type says this isn't optional, so the default won't be invoked and `useWidth` will always return `0` or the result of `ref.current.offsetWidth`.

```suggestion
function FocusArea({width}: Props) {
```